### PR TITLE
✨ UI: Show announcement banner on entity pages and options to manage the announcement

### LINKED ISSUE #6335: UI: Show announcement banner on entity pages and options to manage the announcement

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Closes #6335 

### Describe your changes :

I worked on showing announcement banners on entity pages and options to manage the announcement.
This PR will add support for showing an active announcement banner and announcement drawer. so, we can merge this chunk and test it out and share the feedback.

To add support for edit and delete I will work on separate PR, created ticket for the same https://github.com/open-metadata/OpenMetadata/issues/6572

cc : @chirag-madlani @harshach @vivekratnavel 

#
### Type of change :

- [x] New feature

#
### Frontend Preview (Screenshots) :
image

image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/FeedListBody.tsx [hunk 1]
```diff
@@ -137,6 +137,7 @@ const FeedListBody: FC<FeedListBodyProp> = ({
                   : isAnnouncement
                   ? '1px solid #FFC143'
                   : leftPanelAntCardStyle.border,
+                background: isAnnouncement ? '#fffbf3' : '',
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/FeedListBody.tsx:140]: can we move this colour to constants.ts and use it everywhere?

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityThreadPanel/ActivityThreadList.tsx [hunk 2]
```diff
@@ -101,6 +101,7 @@ const ActivityThreadList: FC<ActivityThreadListProp> = ({
                           : isAnnouncement
                           ? '1px solid #FFC143'
                           : leftPanelAntCardStyle.border,
+                        background: isAnnouncement ? '#fffbf3' : '',
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityThreadPanel/ActivityThreadList.tsx:104]: we can use constant here

File: openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/AnnouncementCard/AnnouncementCard.less [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.announcement-card {
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/AnnouncementCard/AnnouncementCard.less:1]: please add the licence here

**Review by ShaileshParmar11 — COMMENTED** [review]
left a few comments, can we also add a unit test for the new component?

**Comment by Sachin-chaurasiya** [comment]
> left a few comments, can we also add a unit test for the new component?

Thanks, @ShaileshParmar11 , For the review, I will work on the comments and Yes, the unit test will be added in my next patch as still edit and delete part is pending for announcements. https://github.com/open-metadata/OpenMetadata/issues/6572.

File: openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/AnnouncementCard/AnnouncementCard.less (hunk 3 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/AnnouncementCard/AnnouncementCard.less:1]: Do we need that in the CSS file?

Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/AnnouncementCard/AnnouncementCard.less:1]: yes, in every file

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**